kitsune/users/migrations/0034_delete_old_account_events.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations
+from django.utils import timezone
+from datetime import timedelta
+
+
+def delete_old_account_events(apps, schema_editor):
```

Comment:
let's match the name with the one of the management command

Reply:
Done